### ISSUE TITLE
Price/odds don't update after trade succeeds after 5+ min.

### ISSUE THREAD

**Issue by joeykrug** [AugurProject/augur#176]
```[filter] updatePrice: -0x6eeeaf0da5e09ca945a7b4f5a41a7152df6e1ecfd95c6e50186f6e160b34560c app.js:45308:11
trade succeeded: Object { nonce: "0x3", blockHash: "0xcc8479609206b6827b9497768026c7c6b…", blockNumber: "0x2a537", transactionIndex: "0x0", from: "0x2025b185239ba6c972f199a3b5eeb5bfc…", to: "0xcb41186d3fa3eae83aa828eb776a913ff…", value: "0x0", gas: "0x2fd618", gasPrice: "0xba43b7400", input: "0x7d9e76410000000000000000000000000…", 2 more… }```
Price should update instantly upon this log, but it doesn't

**Comment by tinybike** [comment]
I put a 3 second delay on it (to give the cache a moment to update) -- does it not update at all, or just not update instantly?

**Comment by tinybike** [comment]
N/m missed the 5+ minutes part.  Ha.  Troubleshooting now.

**Comment by tinybike** [comment]
Fixed in 46b4fe842f925c37f54f060a85e0d33bba23afb4.
